.nvmrc

```diff
@@ -1 +1 @@
-7.10.0
+8.9.0
```

Comment:
have you tried node 10? Node 8 is EOL soon

Reply:
I picked `8.x.x` but `10.x.x` should also work.